laplace/lllaplace.py

```diff
@@ -113,16 +113,23 @@ def fit(self, train_loader, override=True):
         super().fit(train_loader, override=override)
         self.mean = parameters_to_vector(self.model.last_layer.parameters()).detach()
```

Comment:
@wiseodd Does the `detach()` have to stay here? You removed it for `ParametricLaplace` subclasses (same question as for `LowRankLaplace`).

Reply:
> Yeah I think a class-level attribute is probably the best solution. Also, maybe it makes sense to consistently name the argument which decides wether to detach or not; in some places it's called detach and here enable_backprop.

See [7726b3d](https://github.com/AlexImmer/Laplace/pull/120/commits/7726b3d9ee431e2a6c258ab27ddecd3b60d17b56)

Reply:
I'll update it once I know how to enable backdrop in ASDL since `LowRankLaplace` only supports ASDL.